### ISSUE TITLE
No warnings but no playback on multiple platform with 0.1.0 and latest develop: potential API changes?

### ISSUE THREAD

**Issue by markubiak** [librespot-org/librespot#418]
Hello,

I've been using librespot for years now, but sometime in the last week, my Pi stopped working with librespot. The log shows that the track is loaded after connection, but even when loaded (which seems to take much longer than it used to), the play and pause commands never seem to make it to Librespot. My desktop shows the song as playing, but Librespot does not. When the device transitions to the next song, the desktop client disconnects from Librespot. Again, the server running the binary shows no warnings or information other than "Loading <track>..." and sometimes "Loaded <track>".

I can reproduce this with 0.1.0 built from master, 0.1.0 from crates.io, and the latest dev branch. I also tried running librespot on my desktop (x86_64) installed with cargo, and see an identical issue. This leads me to believe something changed upstream.

This project actually got me to start learning rust, so if someone else can verify it's a code issue, I could be willing to take a look and see if I can take a crack at fixing it. Thanks

**Comment by markubiak** [comment]
Some updates: I can reproduce this on 3 machines, identical behavior for all. On one of the machines, I was also able to run librespot-java successfully and that program does not share this behavior. I've also found that "no playback" is not correct, but there is a 30+ second delay (check out the timestamps). After the song finally starts, it frequently drops out and back in.

I've attached debug logs for librespot and librespot-java for the same flow (host server, play on Android phone, connect to librespot, detach). Same behavior happens when using a Linux desktop client to connect to librespot.

[librespot.log](https://github.com/librespot-org/librespot/files/4018296/librespot.log)
[librespot-java.log](https://github.com/librespot-org/librespot/files/4018297/librespot-java.log)
